Cartfile.resolved

```diff
@@ -1 +1,2 @@
+github "robrix/Box" "1.1.0"
 github "thoughtbot/runes" "v1.2.0"
```

Comment:
This should also have no quotes, and should be `~> 1.2.0` (I know this wasn't your change, but since you're in here)